campaign_info.toml

```diff
@@ -6,24 +6,24 @@
 [desktop]
 name = "Desktop"
 icon = "desktop"
-campaign = "C24_WMDE_Desktop_DE_00"
+campaign = "C24_WMDE_Desktop_DE_01"
 description = "Based on C23_WMDE_Desktop_DE_25"
-campaign_tracking = "00-ba-190624"
+campaign_tracking = "01-ba-010724"
 preview_link = "/wiki/Wikipedia:Hauptseite?devbanner={{banner}}&banner=B22_WMDE_local_prototype"
 preview_url = 'https://de.wikipedia.org/wiki/Wikipedia:Hauptseite?banner={{banner}}&devMode'
 wrapper_template = "wikipedia_org"
 use_of_funds_source = "MediaWiki:WMDE_Fundraising/UseOfFunds_2024_DE"
 
 # Banners of the campaign, key after "banners" can be anything
 [desktop.banners.ctrl]
-filename = "./banners/desktop/C24_WMDE_Desktop_DE_00/banner_ctrl.ts"
-pagename = "B24_WMDE_Desktop_DE_00_ctrl"
-tracking = "org-00-190624-ctrl"
+filename = "./banners/desktop/C24_WMDE_Desktop_DE_01/banner_ctrl.ts"
+pagename = "B24_WMDE_Desktop_DE_01_ctrl"
+tracking = "org-01-010724-ctrl"
 
 [desktop.banners.var]
-filename = "./banners/desktop/C24_WMDE_Desktop_DE_00/banner_var.ts"
-pagename = "B24_WMDE_Desktop_DE_00_var"
-tracking = "org-00-190624-var"
+filename = "./banners/desktop/C24_WMDE_Desktop_DE_01/banner_var.ts"
+pagename = "B24_WMDE_Desktop_DE_01_var"
+tracking = "org-01-010724-var"
```

Comment:
got the info from Kai today that the date format should be yymmdd, I had to change it in the mobile banner too
-> 240701